pages/check_availability.tsx

```diff
@@ -40,6 +40,38 @@ export default function CheckAvailability() {
       });
     }
   };
+//==============================================================================
+// function for getting user district and displaying CVCs
+function getUserLocation() {
+  const success= async(pos)=>{
+    const latitude = pos.coords.latitude;
+    const longitude = pos.coords.longitude;
+    try {
+      const result = await fetch(`https://api.opencagedata.com/geocode/v1/json?q=${latitude}+${longitude}&key=${process.env.NEXT_PUBLIC_GEOCODING_API_KEY}`);
+      const resultJSON = await result.json();
+      const district= resultJSON.results[0].components.county;
```

Comment:
The `country` key returns the country (i.e., India)

I can see there's a field for the postalcode/pincode, can we make the query using that

<img width="1044" alt="Screenshot 2021-05-15 at 9 38 22 PM" src="https://user-images.githubusercontent.com/27439197/118370593-f051a780-b5c5-11eb-86bc-4744fb5153f5.png">